### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by lenkon**
### Project description: 
- This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that will be created using React and Redux. This website list the genres of popular shows and allows to see the shows descriptions.

### Project requirements:
**API**
First you need to choose an API to base the development of the webapp on. The API should allow you to get numeric values based on a parameter. We recommend that you choose an API that is open or needs an API key. If you choose an API that require authentication, **you should implement it on your own**. 

  Some example APIs are:
  - [Financial modeling prep API](https://financialmodelingprep.com/developer/docs/): stock data.
  - [Air pollution API](https://openweathermap.org/api/air-pollution): air quality data.

  You can find many APIs in [this GitHub repo](https://github.com/public-apis/public-apis) or in [ProgrammableWeb](https://www.programmableweb.com/category/all/apis). Some of the APIs requires a token for authentication and some others are just open.

**Design**
- You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
  - Colors (select one main color for your website).
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.

Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).

The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

**Interactions**
- Home page
  - When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
  - Along with the category name, you will display some numeric values per category that come from the API.
  - When the user clicks (or taps) on a category item, the application navigates to the details page.

- Details page
  - In the details page, the webapp retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

**Testing requirements**
- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.
  - You may need to mock the access to the API, so that your tests don't send actual requests.
  - You may need to mock the connection to the Redux Store.

**Technical requirements**

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- You should filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project should be deployed and accessible online using [Render](https://render.com), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/).
  - Add a link to the online version of your app to your README file.

### API Used: 
- TMDB API

### Project requirements link: 
- https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,151 @@
...
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please rate and give a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for sharing the tools and requirements, and TMDB for the APIs used.
```
Comment (ghost) [inline README.md]: - [x] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @lenkon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App is tested properly
✅ Good design 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lenkon** [comment]
Hi @frank1738 ,

Thanks for your valuable feedback. I have added the appropriate credits to the creator of the original design idea.

Best Regards,
K. Inoulen Shanongba

**Review by binhussen — APPROVED** [review]
Hi @lenkon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Used react-redux and API ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**